public/js/helpers/humanize_helper.js

```diff
@@ -138,25 +138,31 @@ var humanize = {
       }
       return result
     }
+    var pad = function (x) {
+      if (x < 10) {
+        return `&nbsp${x}`
```

Comment:
This is actually showing "&nbsp" on the page.  Even with a `;`, it still doesn't work.  I don't know what the problem is with that, but why not use `padStart` with a unicode non-breaking space:

```js
x.toString().padStart(2, '\u00a0')
```